src/dashboard/Push/PushDetails.react.js

```diff
@@ -39,7 +39,13 @@ const EXP_STATS_URL = 'http://docs.parseplatform.org/ios/guide/#push-experiments
 let getMessage = (payload) => {
   if(payload) {
     let payloadJSON = JSON.parse(payload);
-    return payloadJSON.alert ? payloadJSON.alert : payload;
+		if (payloadJSON.alert.body) {
```

Comment:
@repertus Robert, we wanted to make a release in the next few days - if you could do these changes, it will enter in the next release.